test/e2e/kubelet_perf.go

```diff
@@ -216,19 +217,9 @@ var _ = Describe("Kubelet [Serial] [Slow]", func() {
 					"/docker-daemon": &containerResourceUsage{MemoryRSSInBytes: 85 * 1024 * 1024},
 				},
 			},
-			// TODO(yujuhong): change this test to ~100 pods per node after
-			// --max-pods have been changed.
-			{podsPerNode: 35,
-				cpuLimits: containersCPUSummary{
-					"/kubelet":       {0.50: 0.12, 0.95: 0.14},
-					"/docker-daemon": {0.50: 0.06, 0.95: 0.08},
-				},
-				// We set the memory limits generously because the distribution
-				// of the addon pods affect the memory usage on each node.
-				memLimits: resourceUsagePerContainer{
-					"/kubelet":       &containerResourceUsage{MemoryRSSInBytes: 75 * 1024 * 1024},
-					"/docker-daemon": &containerResourceUsage{MemoryRSSInBytes: 100 * 1024 * 1024},
-				},
+			// TODO(yujuhong): Gather numbers and set the limits.
```

Comment:
Every test takes 20 min, so this will increase the overall test time for the serial suite. If we're okay with that, I'll keep the original.